src/dlgConnectionProfiles.cpp

```diff
@@ -133,8 +131,10 @@ void dlgConnectionProfiles::slot_update_description()
     if( pItem )
     {
         QString profile = pItem->text();
-        QString desc = mud_description_textedit->toPlainText();
-        writeProfileData( profile, QStringLiteral( "description" ), desc );
+        QString description = mud_description_textedit->toPlainText();
+        writeProfileData( profile, QStringLiteral( "description" ), description );
+
+        // don't display custom profile descriptions as a tooltip, as passwords could be stored in there
```

Comment:
Not a helpful comment IMHO - If the user _wants_ to write the password in the description area isn't that _their_ problem - after all it _will_ be shown if they click on the icon to select the profile to play it! 🙁